app/assets/stylesheets/darkswarm/shop_tabs.css.scss

```diff
@@ -7,7 +7,7 @@
 
   .tab-buttons {
     color: $dark-grey;
-    box-shadow: 0 1px 0 $light-grey-transparency;
+    box-shadow: 0 1px 0 rgba(0,0,0,0.05), 0 8px 6px -6px rgba(0, 0, 0, 0.2);
```

Comment:
these should be colors with names.

Reply:
It's a very specific shadow effect on a single element that I don't think will be used anywhere else. Does it really need need two new colour variables added to the list?

Reply:
sorry, I missed that it was the 4th param...